.github/workflows/release.yml

```diff
@@ -50,18 +52,9 @@ jobs:
       - name: PDK Init
         uses: ./.github/actions/pdk-init
       - name: Build
-        run: NX_CLOUD_NO_TIMEOUTS=true pnpm build
+        run: pnpm build
       - name: Release
         run: pnpm release
-      - name: Check for mutations
-        run: git diff --ignore-space-at-eol --exit-code
-      - name: Tag last-release
-        shell: bash
-        run: |
-          git tag -f last-release
-          git push origin last-release --force
-      - name: Print release tags
-        run: git tag --list --contains `git rev-parse HEAD`
```

Comment:
- Removed "check for mutations" because we still want to push docs if nothing was releases.
- The "Tag last-release" is already handled in the "release" task, so removing it here as was duplicate
- Moved "Print release tags" to the release task - just output for all the release tags, not important